web/sites/default/config/field.field.group.advisory_group.field_email.yml

```diff
@@ -16,7 +16,7 @@ field_name: field_email
 entity_type: group
 bundle: advisory_group
 label: Email
-description: ''
+description: 'DEPRECATED. This information is being moved to the <strong>contact</strong> field. '
```

Comment:
Is it safe to remove this field?

Reply:
Not yet. I'm marking the fields as deprecated so that we can help editors migrate to the new contacts field. Once all the data is migrated manually, we can display the new field instead of the old field, then finally remove the deprecated fields.

Reply:
Sounds good.